### PR TITLE
add manual cors whitelist

### PR DESCRIPTION
**Pull request by klawani**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/ROC-5274
https://tools.hmcts.net/jira/browse/RSE-130

### Change description ###

Add CORS whitelist manually to check if it resolves this issue:

_(index):1 Access to XMLHttpRequest at 'https://gateway-ccd-cmc-citizen-frontend.demo.platform.hmcts.net/data/internal/profile' from origin 'https://case-management-web-ccd-cmc-citizen-frontend.demo.platform.hmcts.net' has been blocked by CORS policy: Response to preflight request doesn't pass access control check: Redirect is not allowed for a preflight request.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x ] No
```

### CODE REVIEW

File: k8s/demo/common/money-claims/cmc-demo.yaml [hunk 1]
```diff
@@ -94,6 +94,7 @@ spec:
             fromSecret: true
             secretKeyRefName: ccd-api-gateway-oauth2-client-secret
             secretKeyRefKey: key
+            CORS_ORIGIN_WHITELIST: https://case-management-web-ccd-cmc-citizen-frontend.demo.platform.hmcts.net
```
Comment (timja) [inline k8s/demo/common/money-claims/cmc-demo.yaml]: also shouldn't this be under environment?
pretty sure you can just put in *, but do that later

Comment (timja) [reply k8s/demo/common/money-claims/cmc-demo.yaml]: This already seems to be set to this:
```
 CORS_ORIGIN_METHODS:             GET,POST,OPTIONS,PUT,DELETE
 CORS_ORIGIN_WHITELIST:           *
```

Comment (klawani) [reply k8s/demo/common/money-claims/cmc-demo.yaml]: Yeah ... But I thought perhaps CORS doesn't like the * ... hence I wanted to try being specific.

Comment (timja) [reply k8s/demo/common/money-claims/cmc-demo.yaml]: I can fix it, will send PRs on monday I know what to do...

Comment (klawani) [reply k8s/demo/common/money-claims/cmc-demo.yaml]: Thanks Tim ...

**Pull request closed without merging**